docs/source/kedro_project_setup/configuration.md

```diff
@@ -12,6 +12,39 @@ You can also specify a source directory for the configuration files at run time
 ```bash
 kedro run --conf-source=<path-to-new-conf-directory>
 ```
+
+If you're using the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) you can also read configuration from a compressed file in `zip` or `tar` format. The two commands below are examples of how to reference a `tar.gz` or `zip` file:
+
+ ```bash
+kedro run --conf-source=<path-to-compressed-file>.tar.gz
+
+kedro run --conf-source=<path-to-compressed-file>.zip
+```
+
+To compress your configuration you can either leverage Kedro's `kedro package` command which builds the package into the `dist/` folder of your project, and creates one `.egg` file, one `.whl` file, as well as a `tar.gz` file containing the project configuration. This compressed version of the config files excludes any files inside your `local` directory.
+Alternatively you can run the command below to create a `tar.gz` file:
```

Comment:
Any files contained in local are meant to stay local and not be distributed. So when you're packaging your project to deploy or share, you should ensure that credentials and other sensitive data is provided in a appropriate way e.g. through a vault or environment variables. 

If the `credentials.yml` file doesn't contain any sensitive data and is using environment variables a user could move it to another (not local) environment and then it would be packaged up. I'm not sure we should advice this, because people might still accidentally move sensitive data. So I'd suggest for now to just leave it as is, because users at the moment need to also use something else from a `credentials.yml` if running in production. (Of course it could be they're not, which would be bad, but I don't think we can do more other than advice what the best way would be.)